Makefile

```diff
@@ -1,15 +1,31 @@
 .PHONY: help format install lint test
 
-help:
-	@grep '^\.PHONY' Makefile | cut -d' ' -f2- | tr ' ' '\n'
+help:  ## ⁉️  - Display help comments for each make command
+	@grep -E '^[0-9a-zA-Z_-]+:.*? .*$$'  \
+		$(MAKEFILE_LIST)  \
+		| awk 'BEGIN { FS=":.*?## " }; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'  \
+		| sort
 
-install:
+install:  ## ⚙️  - Install with testing dependencies, and pre-commit
+	@echo "⚙️ - Running install in editable mode with testing dependencies"
 	python -m pip install -e ".[testing]"
 	python -m pip install -U pre-commit
 	pre-commit install
 
-lint:
+lint:  ## 🧹  - Lint and format
+	@echo "🧹 - Running pre-commit to lint and format"
 	git ls-files --others --cached --exclude-standard | xargs pre-commit run --files
 
-test:
-	coverage run testmanage.py test --deprecation all
+test:  ## 🧪  - Run tests
+	@echo "🧪 - Running test suite"
+	tox
+
+clean:	## 🗑️  - Remove __pycache__ and test artifacts
+	@echo "🗑️ - Removing __pycache__ and test artifacts"
+	find . -name ".tox" -prune -o -type d -name  "__pycache__" -exec rm -r {} +
+
+package-setup:
+	@echo "📦 - Packaging for PyPI"
+	flit build --setup-py
```

Comment:
You're running `flit build --setup-py` here (which will generate a package with both a `pyproject.toml` and also a generated `setup.py` for backwards compatibility with old versions of `pip` - less and less necessary as time goes on) but then in the release action you're running `flit build` (which will generate a package with `pyproject.toml` only).
1. Lets pick one or the other and do it consistently
2. If we're going to have a make command with an abstraction over this, shall we use it in the GH action too?

Reply:
need to go and check why I did it this way 😂
imho, we should require a more recent version of pip, and just do `flit build`

Reply:
Yeah, so I think for newly created packages we should just `flit build` (which is why I've gone with that on the cookiecutter repo). I reckon at the time we migrated wagtail-localize, we probably wanted the backwards compatibility. We are/were still maintaining compatibility with some old python/django/wagtail versions. Arguably dropping the ability to install with pip<=somenumber is a breaking change for older packages.